.github/workflows/vm-pr.yml

```diff
@@ -98,15 +131,52 @@ jobs:
         
         # Tests were splitted with --dir and --excludeDir to balance execution times below the 9min mark.
         args: [
-          # '--fork=Berlin --expected-test-amount=33',
           '--fork=Istanbul --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          '--fork=Istanbul --excludeDir=stTimeConsuming --expected-test-amount=19817',
-          # '--fork=Constantinople --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          # '--fork=Constantinople --excludeDir=stTimeConsuming --expected-test-amount=17193',
-          # '--fork=Petersburg --dir=GeneralStateTests/stTimeConsuming --expected-test-amount=15561',
-          # '--fork=Petersburg --excludeDir=stTimeConsuming --expected-test-amount=17174',
-          # '--fork=Homestead --expected-test-amount=6997',
-          # '--fork=Chainstart --expected-test-amount=4385'
+          '--fork=Istanbul --excludeDir=stTimeConsuming --expected-test-amount=19817'
+        ]
+      fail-fast: false
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: actions/checkout@v1
+
+      - name: Dependency cache
+        uses: actions/cache@v2
+        id: cache
+        with:
+          key: VM-${{ runner.os }}-12-${{ hashFiles('**/package-lock.json') }}
+          path: '**/node_modules'
+
+      # Installs root dependencies, ignoring Bootstrap All script.
+      # Bootstraps the current package only
+      - run: npm install --ignore-scripts && npm run bootstrap:vm
+        if: steps.cache.outputs.cache-hit != 'true'
+        working-directory: ${{github.workspace}}
+
+      # Builds current package and the ones it depends from.
+      - run: npm run build:vm
+        working-directory: ${{github.workspace}}
+
+      - run: npm run test:blockchain -- ${{ matrix.args }}
+
+  vm-blockchain-extended:
+    if: contains(join(github.event.pull_request.labels.*.name, ' '), 'Test all hardforks')
```

Comment:
coool

Reply:
I could not use the entire label name `type: Test all hardforks`, because YAML libraries have trouble with character escaping. The colon was interpreted as a regular `key: value`, resulting in syntax errors. 🤷‍♂️

Reply:
interesting, ok, good to know! and I noticed the label has a lowercase t for test.